CoreHistogramView/src/au/gov/asd/tac/constellation/views/histogram/HistogramDisplay.java

```diff
@@ -339,9 +341,16 @@ public void paintComponent(Graphics g) {
 
                 final String headerStringValue = getStringToFit(PROPERTY_VALUE, textWidth, g2);
                 final String headerStringCount = getStringToFit(COUNT, barsWidth, g2);
-                g2.drawString(headerStringValue, LEFT_MARGIN + iconPadding, TOP_MARGIN + (barHeight / 2) + correction);
-                g2.drawString(headerStringCount, barLeft, TOP_MARGIN + (barHeight / 2) + correction);
-
+                final String headerStringTotalBins = getStringToFit(TOTAL_BINS_COUNT + bins.length, barsWidth, g2);
+                
+                int countTextWidth = g2.getFontMetrics().stringWidth(headerStringTotalBins);
+                
+                g2.drawString(headerStringValue, LEFT_MARGIN + iconPadding, 
+                        TOP_MARGIN + (barHeight / HALF) + correction);
+                g2.drawString(headerStringCount, barLeft, TOP_MARGIN + (barHeight / HALF) + correction);
+                g2.drawString(headerStringTotalBins, getParent().getWidth() - countTextWidth, 
+                        TOP_MARGIN + (barHeight /HALF) + correction);
+                
```

Comment:
@antares1470 I find the rule about being more than 120 chars too strict and as a result its making it harder to read for me when its split into 2 lines like this. If you agree can we omit that SonarCloud rule for now?

@Delphinus8821 don't worry about changing this back.